app/views/bars/show.html.erb

```diff
@@ -37,14 +40,24 @@
     </div>
   </div>
 
-<div>
   <% @reviews.order(created_at: :desc).each do |review| %>
-    <div><%= review.title + " " + review.rating.to_s + " " + review.body %></div>
+    <div class="review">
+      <h3><%= review.title %></h3>
+      <h4><%= review.rating.to_s %>/5</h4>
```

Comment:
`<%= "#{review.rating}" %>`

Though, why do you need this at all? Would `<%= review.rating %>` work fine?